obspy/io/reftek/tests/test_core.py

```diff
@@ -22,6 +22,16 @@ class ReftekTestCase(unittest.TestCase):
     Test suite for obspy.io.reftek
     """
     def setUp(self):
+        try:
+            # doctests of __init__.py produce warnings that get caught. if we
+            # don't raze the slate out the registry here, we can't test those
+            # warnings in the unit tests (if doctests run before unittests)..
```

Comment:
Which warnings do you mean? If you're referring to line 381; that block is missing the `warnings.simplefilter('always')` that is in every other one.